test/liquid_voting/delegations_test.exs

```diff
@@ -108,6 +118,16 @@ defmodule LiquidVoting.DelegationsTest do
       assert {:error, %Ecto.Changeset{}} = Delegations.create_delegation(args)
     end
 
+    # FAILING!!
+    # PR #109 https://github.com/liquidvotingio/api/issues/109
+    # "Allow creation of more than one proposal-specific delegation (different proposals) with same delegator and delegate"
+    # causes this failure.
+    # Addressed in issue #125:https://github.com/liquidvotingio/api/issues/125
+    # This test case should actually be acceptable (should not return a changeset error), though we need to ensure (in applying
+    # issue #125, above) that the proposal-specific delegation takes precedence (to prevent double delegation counting if delegate votes)
+    # Thus, delete this test and add a new test of proposal-specific delegation takes precedence over global delegation for same
+    # delegator/delegate pair, when issue #125 is addressed.
```

Comment:
Again, I might be missing something here, but - seems to me this test is still current, since if a global delegation is present, and proposal-specific one for the same delegate is redundant. In #125 we'll only allow proposal-specific ones for _different_ delegates, and those will indeed be the only ones counted when a global delegatoin for _another_ delegate is also present.

Reply:
You mean that number 1, we agreed in #125: "If a proposal-specific delegation is created, then an existing global delegation for same delegator/delegate pair should persist, but proposal-specific delegation takes precedence (when voting result is calculated)."  
Is not quite right?  
Instead, we should prevent creation of a proposal-specific delegation, if there is an existing global delegation for same delegator/delegate pair?

Reply:
Exactly.